packages/expo-image-picker/ios/UIImagePickerControllerExtentsion.swift

```diff
@@ -0,0 +1,58 @@
+// Copyright 2016-present 650 Industries. All rights reserved.
+
+extension UIImagePickerController {
+  func fixCannotMoveEditingBox() {
+    if let cropView = cropView,
+      let scrollView = scrollView,
+      scrollView.contentOffset.y == 0
+    {
+      let top = cropView.frame.minY + self.view.safeAreaInsets.top
```

Comment:
Hi @ebrahimhassan121
adding the safeAreaInsets.top prevents from reaching the bottom, and overflows the top
![79758501-49ED-4661-A575-EBF0DC0A29AD_1_101_o](https://github.com/expo/expo/assets/20543676/31e108a2-038a-4c2b-a291-74c2adbe3b84)
![18FCFD88-330C-4006-BEB6-36A287F2A96E_1_101_o](https://github.com/expo/expo/assets/20543676/7e32cbff-aae2-49e6-8ee1-3a72295f7ef3)